packages/core/saved-objects/core-saved-objects-server-internal/src/routes/bulk_create.ts

```diff
@@ -55,6 +55,16 @@ export const registerBulkCreateRoute = (
       usageStatsClient.incrementSavedObjectsBulkCreate({ request: req }).catch(() => {});
 
       const { savedObjects } = await context.core;
+
+      const unsupportedTypes = [...new Set(req.body.map(({ type }) => type))].filter((tname) => {
+        const fullType = savedObjects.typeRegistry.getType(tname);
+        if (!fullType?.hidden && fullType?.hiddenFromHttpApis) {
+          return fullType.name;
+        }
+      });
+      if (unsupportedTypes.length > 0) {
+        throwOnHttpHiddenTypes(unsupportedTypes);
+      }
```

Comment:
yea, it can but I was trying to keep the testing overhead to a minimum. Extracting the method to `utils.ts` means we need to update the jest tests to ock the repository.

Reply:
I went ahead and made the change: [`8a034af` (#149166)](https://github.com/elastic/kibana/pull/149166/commits/8a034af81187a30f6103808bfe12f5cff5cb4a65)